product_docs/docs/pgd/5.6/commit-scopes/commit-scopes.mdx

```diff
@@ -175,4 +175,52 @@ SELECT bdr.alter_node_group_option(
 );
 ```
 
+## Degrade options
 
+CAMO, SYNCHRONOUS_COMMIT, and GROUP COMMIT each have rule degrade options.
```

Comment:
You need to explain what the degrade option enables, I can see no preceding mention of it. And consider whether this really needs its own page so it can be easily referenced from elsewhere.

Some discussion of the process of degrading is appropriate now its expanded so much.

"When a node applies a transaction and that transaction times out, it could be useful trigger a process of degrading the requirements of the transaction to be completed, rather than just rolling back. DEGRADE ON offers a route to this capability to gracefully degrade a transaction. At its simplest, DEGRADE ON and take a timeout and a second set of commit scope operation for the commit scope to gracefully degrade to. In the case of GROUP COMMIT and SYNCHRONOUS_COMMIT this can be a fully specified operation, eg, an ALL (group) MAJORITY can degrade to an ANY (group) MAJORITY"

And then you need some explainer around why you would be concerned about the triggering transaction having or not having come from the write leader.

And a mention of the scope of the degrade (I believe it ends at the end of the transaction, but I may be wrong).

Reply:
1. I added something along the lines of what you included in your suggested copy.
2. 

> And then you need some explainer around why you would be concerned about the triggering transaction having or not having come from the write leader.

But why would I be concerned about the triggering transaction having or not having come from the write leader?

3. Isn't the scope wider than the single transaction given that (at least the way I am understanding it) other transactions wouldn't have to wait their timeouts to degrade if it was determined by the connectivity checker that their timeout would have been satisfied:

> ...the PGD manager process also periodically (every 5s) checks the connectivity and apply rate (the one in [bdr.node_replication_rates](/pgd/latest/reference/catalogs-visible/#bdrnode_replication_rates)) and if there are commit scopes that would degrade at that point based on the current state of replication, they will be automatically degraded—such that any transaction processing at that time uses the degraded rule instead of waiting for timeout.

Reply:
Re 2: That's something we need to establish. If there's an option, there's a reason for the option. I assume its so you only degrade when the write leader is having issues and you aren't concerned with forcing a degrade from the non-write leader nodes.

3. Move the behavior section up to the general introduction.

Reply:
2. Okay I got some feedback from Petr and tried to incorporate this. 
3. done.
4. What is the action item here? I added sql tags to the blocks, but since turning wrapping on is manual I am not sure what to do with this.